tools/pom.xml

```diff
@@ -100,9 +105,16 @@
                             <artifactSet>
                                 <includes>
                                     <include>org.rogach:scallop_${scala.binary.version}</include>
+                                    <include>com.nvidia:rapids-4-spark-common_${scala.binary.version}</include>
                                 </includes>
                             </artifactSet>
                             <filters>
+                                <filter>
+                                    <artifact>com.nvidia:rapids-4-spark-common_${scala.binary.version}</artifact>
+                                    <includes>
+                                        <include>com/nvidia/spark/rapids/ThreadFactoryBuilder.class</include>
```

Comment:
it looks like you need to include more then just the .class, perhaps use a * at the end.

```
Exception in thread "main" java.lang.NoClassDefFoundError: com/nvidia/spark/rapids/ThreadFactoryBuilder$$anon$1
        at com.nvidia.spark.rapids.ThreadFactoryBuilder.build(ThreadFactoryBuilder.scala:41)
        at com.nvidia.spark.rapids.tool.profiling.Profiler.<init>(Profiler.scala:40)
        at com.nvidia.spark.rapids.tool.profiling.ProfileMain$.mainInternal(ProfileMain.scala:56)
        at com.nvidia.spark.rapids.tool.profiling.ProfileMain$.main(ProfileMain.scala:32)
        at com.nvidia.spark.rapids.tool.profiling.ProfileMain.main(ProfileMain.scala)
Caused by: java.lang.ClassNotFoundException: com.nvidia.spark.rapids.ThreadFactoryBuilder$$anon$1
        at java.net.URLClassLoader.findClass(URLClassLoader.java:387)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
        ... 5 more
```

Reply:
Added * at the end. Looks like the failure is unrelated to this change.


`
17:57:38  [ERROR] Failed to execute goal on project rapids-4-spark-sql_2.12: Could not resolve dependencies for project com.nvidia:rapids-4-spark-sql_2.12:jar:22.06.0-SNAPSHOT: Failed to collect dependencies at org.apache.spark:spark-avro_2.12:jar:3.1.1.3.1.7270.0-253: Failed to read artifact descriptor for org.apache.spark:spark-avro_2.12:jar:3.1.1.3.1.7270.0-253: Could not transfer artifact org.apache.spark:spark-avro_2.12:pom:3.1.1.3.1.7270.0-253 from/to cloudera-repo (https://repository.cloudera.com/artifactory/cloudera-repos/): PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target -> [Help 1]
`